pyproject.toml

```diff
@@ -25,3 +25,7 @@ known-first-party = ["primap2"]
 convention = "numpy"
 
 [tool.setuptools_scm]
+
+[tool.pytest.ini_options]
+# actually, importlib.resources.open_text will be undeprecated again in python 3.13
```

Comment:
Crazy

Reply:
Yes, it was actually scheduled for removal in python 3.13, but at the last minute someone contributed a solution which made it possible to maintain the function going forward instead.